README.md

```diff
@@ -1,188 +1,81 @@
-
-From the _root_ directory:
-```
-pip install seqrepo
-sudo mkdir /usr/local/share/seqrepo
-sudo chown $USER /usr/local/share/seqrepo
-seqrepo pull -i 2021-01-29  # Replace with latest version using `seqrepo list-remote-instances` if outdated
-```
-
-If you get an error similar to the one below:
-```
-PermissionError: [Error 13] Permission denied: '/usr/local/share/seqrepo/2021-01-29._fkuefgd' -> '/usr/local/share/seqrepo/2021-01-29'
-```
-
-You will want to do the following:\
-(*Might not be ._fkuefgd, so replace with your error message path*)
-
-```console
-sudo mv /usr/local/share/seqrepo/2021-01-29._fkuefgd /usr/local/share/seqrepo/2021-01-29
-```
-
-Use the `SEQREPO_ROOT_DIR` environment variable to set the path of an already existing SeqRepo directory. The default is `/usr/local/share/seqrepo/latest`.
-
-### Database Initialization
-
-The Normalizer supports two data storage options:
-
-* [DynamoDB](https://aws.amazon.com/dynamodb), a NoSQL service provided by AWS. This is our preferred storage solution. In addition to cloud deployment, Amazon also provides a tool for local service, which can be installed [here](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DynamoDBLocal.DownloadingAndRunning.html). Once downloaded, you can start service by running `java -Djava.library.path=./DynamoDBLocal_lib -jar DynamoDBLocal.jar -sharedDb` in a terminal (add a `-port <VALUE>` option to use a different port)
-* [PostgreSQL](https://www.postgresql.org/), a well-known relational database technology. Once starting the Postgres server process, [ensure that a database is created](https://www.postgresql.org/docs/current/sql-createdatabase.html) (we typically name ours `gene_normalizer`).
-
-By default, the Gene Normalizer expects to find a DynamoDB instance listening at `http://localhost:8000`. Alternative locations can be specified in two ways:
-
-The first way is to set the `--db_url` option to the URL endpoint.
+**[Live service](https://normalize.cancervariants.org/gene)**
 
-```commandline
-gene_update --update_all --db_url="http://localhost:8001"
-```
-
-The second way is to set the `GENE_NORM_DB_URL` environment variable to the URL endpoint.
-```commandline
-export GENE_NORM_DB_URL="http://localhost:8001"
-```
-
-To use a PostgreSQL instance instead of DynamoDB, provide a PostgreSQL connection URL instead, e.g.
-
-```commandline
-export GENE_NORM_DB_URL="postgresql://postgres@localhost:5432/gene_normalizer"
-```
-
-### Adding and refreshing data
-
-Use the `gene_update` command in a shell to update the database.
+**[Documentation](https://gene-normalizer.readthedocs.io/en/latest/.html)** · [Installation](https://gene-normalizer.readthedocs.io/en/latest/install.html) · [Usage](https://gene-normalizer.readthedocs.io/en/latest/usage.html) · [API reference](https://gene-normalizer.readthedocs.io/en/latest/api/api.html)
```

Comment:
```suggestion
**[Documentation](https://gene-normalizer.readthedocs.io/en/latest/)** · [Installation](https://gene-normalizer.readthedocs.io/en/latest/install.html) · [Usage](https://gene-normalizer.readthedocs.io/en/latest/usage.html) · [API reference](https://gene-normalizer.readthedocs.io/en/latest/api/api.html)
```

Reply:
Are we having two different RTD versions? Maybe we should point to the 2-alpha release rather than latest

Reply:
You can point to individual versions in RTD (they're keyed off of the python package version, so eg `https://gene-normalizer.readthedocs.io/en/latest/0.3.0-dev0/`) but it's a little awkward  and requires manual updating. I think most people just default point to one of the static endpoints, or operate an entirely different RTD project for different major versions (eg psycopg2 vs 3).